<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1365</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>执行方式:单步执行</string>
    </property>
   </widget>
   <widget class="QComboBox" name="associate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>直接映射</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>两路组相联</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四路组相联</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="cpu1_cache">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>读/写</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标地址</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="cpu2_cache">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>读/写</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标地址</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="cpu3_cache">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>读/写</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标地址</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="cpu4_cache">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>160</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>读/写</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目标地址</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="memory1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Memory1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="memory2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>231</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Memory2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="memory3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>231</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>23</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Memory3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="memory4">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>320</y>
      <width>231</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>24</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>25</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>26</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>27</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>28</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>29</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>31</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Memory4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="cpu1_exec">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>访问地址</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cpu1_read">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>读</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>写</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="cpu1_addr">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>访问地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpu2_addr">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cpu2_exec">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cpu2_read">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>读</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>写</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>访问地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpu3_addr">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cpu3_exec">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>110</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cpu3_read">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>110</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>读</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>写</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>访问地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpu4_addr">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cpu4_exec">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>110</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cpu4_read">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>110</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>读</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>写</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Process1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Process2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Process3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Process4</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1365</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
